kata-webhook/webhook-check.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run this script to check the webhook is deployed and working
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+webhook_dir=$(dirname $0)
+source "${webhook_dir}/../lib/common.bash"
+source "${webhook_dir}/common.bash"
+
+readonly hello_pod="hello-kata-webhook"
+# The Pod RuntimeClassName for Kata Containers.
+RUNTIME_CLASS="${RUNTIME_CLASS:-"kata"}"
+
+cleanup() {
+	{
+	kubectl get -n ${WEBHOOK_NS} pod/${hello_pod} && \
+		kubectl delete -n ${WEBHOOK_NS} pod/${hello_pod}
+	} &>/dev/null
+}
+trap cleanup EXIT
+
+# Check the deployment exists and is available.
+#
+check_deployed() {
+	kubectl get -n ${WEBHOOK_NS} deployment/${WEBHOOK_SVC} &>/dev/null || \
+		die "The ${WEBHOOK_SVC} deployment does not exist"
+
+	kubectl wait -n ${WEBHOOK_NS} deployment/${WEBHOOK_SVC} \
+		--for condition=Available --timeout 60s &>/dev/null || \
+		die "The ${WEBHOOK_SVC} deployment is unavailable after 60s waiting"
```

Comment:
Nit: Hard-coded value in string. I'd consider a `local timeout_secs=60` or similar to be used for `kubectl` and `die`.